hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/OnDemandContainerDataScanner.java

```diff
@@ -128,6 +128,7 @@ private static void performOnDemandScan(Container<?> container) {
       return;
     }
 
+    long startTime = System.nanoTime();
```

Comment:
Let's use `Instant.now().toEpochMillis()` to be consistent with the rest of the code. I think `millis` will be precise enough.